services/web/server/src/simcore_service_webserver/director/director_api.py

```diff
@@ -90,11 +89,14 @@ async def stop_service(
     # stopping a service can take a lot of time
     # bumping the stop command timeout to 1 hour
     # this will allow to sava bigger datasets from the services
+
+    settings: DirectorSettings = get_plugin_settings(app)
```

Comment:
question, are we still using the director-v0 api directly? I thought everything was going through the director-v2 now?

Reply:
I think so ... that is why I added a deprecation warning. 
We can check and totally remove if necessary, but I suggest to do it in a different PR